nxdrive/gui/api.py

```diff
@@ -443,6 +444,51 @@ def default_server_url_value(self) -> str:
         """Make daily job better for our developers :)"""
         return getenv("NXDRIVE_TEST_NUXEO_URL", "")
 
+    @pyqtSlot(str, int, result=list)
+    def disk_space_info_to_width(self, uid: str, total_width: int) -> List[float]:
+        engine = self._manager.engines.get(uid)
+        if not engine:
+            return [0, 0, 0]
+
+        local_sync_size = engine.disk_space.local_sync_files
```

Comment:
Here you are calling 3 times `engine.disk_space`, this is not efficient.
You could store the result into the `space` variable instead and use it directly for computations (remember: short descriptive names are preferable :)).

In fact you could do it with verbosity and yet keeping the code readable, something like that:
```python
space = engine.disk_space
used_without_synced = space.free - space.synced
return (
    space.free * width / space.total,
    used_without_synced * width / space.total,
    space.synced * width / space.total,
)
```
I renamed `total_width` -> `width` also.